public/main.css

```diff
@@ -694,6 +733,18 @@ tbody {
     font-size: 18px;
   }
 
+  /* The actual popup (appears on top) */
+  .popup .popuptext {
+    left: auto;
+    right: -40px;
+  }
+
+  /* Popup arrow */
+  .popup .popuptext::after {
+    left: auto;
+    right: 36px;
+  }
```

Comment:
I think the popup should move to the left earlier. It causes horizontal scrolling at any screen between widths 768px and 920px.